vllm/entrypoints/openai/serving_chat.py

```diff
@@ -33,24 +38,27 @@
                           log_tracing_disabled_warning)
 from vllm.utils import iterate_with_cancellation, random_uuid
 
+env = Environment(loader=FileSystemLoader('./'),
+                  autoescape=select_autoescape())
```

Comment:
Is this unused? If not, the "./" path is scaring me

Reply:
yes, this is unused - thanks for the catch!